src/plugins/intel_cpu/src/emitters/plugin/riscv64/jit_eltwise_emitters.cpp

```diff
@@ -801,7 +801,71 @@ std::set<std::vector<element::Type>> jit_floor_emitter::get_supported_precisions
     [[maybe_unused]] const std::shared_ptr<ov::Node>& node) {
     return {{element::f32}};
 }
+/// FLOOR MOD ///
+jit_floor_mod_emitter::jit_floor_mod_emitter(jit_generator_t* host, cpu_isa_t host_isa, element::Type exec_prc)
+    : jit_emitter(host, host_isa, exec_prc) {
+    prepare_table();
+}
+
+jit_floor_mod_emitter::jit_floor_mod_emitter(jit_generator_t* host,
+                                             cpu_isa_t host_isa,
+                                             const std::shared_ptr<ov::Node>& node)
+    : jit_emitter(host, host_isa, get_arithmetic_binary_exec_precision(node)) {
+    prepare_table();
+}
+
+size_t jit_floor_mod_emitter::get_inputs_num() const {
+    return 2;
+}
+
+size_t jit_floor_mod_emitter::aux_vecs_count() const {
+    return 2;
+}
+
+size_t jit_floor_mod_emitter::aux_fp_gprs_count() const {
+    return 2;
+}
+
+void jit_floor_mod_emitter::emit_impl(const std::vector<size_t>& in_vec_idxs,
+                                      const std::vector<size_t>& out_vec_idxs) const {
+    if (host_isa_ == ov::intel_cpu::riscv64::cpu_isa_t::gv) {
+        emit_isa<ov::intel_cpu::riscv64::cpu_isa_t::gv>(in_vec_idxs, out_vec_idxs);
+    } else {
+        OPENVINO_THROW("Can't create jit eltwise kernel for FLOOR_MOD");
+    }
+}
+template <cpu_isa_t isa>
+void jit_floor_mod_emitter::emit_isa(const std::vector<size_t>& in_vec_idxs,
+                                     const std::vector<size_t>& out_vec_idxs) const {
+    OPENVINO_ASSERT(exec_prc_ == element::f32, "JIT Floor Mod emitter supports only f32 precision");
+
+    const VReg src0 = VReg(in_vec_idxs[0]);
+    const VReg src1 = VReg(in_vec_idxs[1]);
+    const VReg dst = VReg(out_vec_idxs[0]);
+    const VReg tmp1 = VReg(aux_vec_idxs[0]);
+    const VReg tmp2 = VReg(aux_vec_idxs[1]);
+    auto fzero = FReg(aux_fp_gpr_idxs[0]);
+    auto fone = FReg(aux_fp_gpr_idxs[1]);
+
+    h->fmv_w_x(fzero, zero);
```

Comment:
Could you please remove this line - it looks like unused?
Please don't forget to update the method  `aux_fp_gprs_count()` 😊